docs/advanced/api_examples.yml

```diff
@@ -245,7 +245,8 @@
     "get state datasets"
 
   doc: |
-    Get a list of all datasets in a deployment.
+    Get a list of all datasets in a deployment. The filesystem path where
+    the dataset is mounted will be included.
```

Comment:
Lots of other stuff is included too.  What's the motivation behind singling this property out?  Is it to do with the fact that the schema definition is currently unclear about where the path property is actually valid/possible?